### PR TITLE
Upgrading Gatling to 3.9.0 and dependencies to support

### PR DESCRIPTION
**Pull request by jonsoloway-hmcts**
### JIRA link (if applicable) ###

N/A

### Change description ###

Upgrade Gatling to 3.9.0, along with all dependencies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/ccdcaseactivity.scala [hunk 1]
```diff
@@ -37,7 +36,7 @@ val CDSGetRequest =
       .exitHereIfFailed
 
   .exec(http("GetIdamToken")
```
Comment (jmcadam) [inline src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/ccdcaseactivity.scala:38]: We could get rid of this and use the "IdamLogin" file?

File: src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/elasticsearch.scala [hunk 2]
```diff
@@ -1,21 +1,19 @@
-package uk.gov.hmcts.ccd.corecasedata.scenarios
+package scenarios.api
```
Comment (jmcadam) [inline src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/elasticsearch.scala:1]: This file is used in the test yet has been deleted?

**Review by jmcadam — CHANGES_REQUESTED** [review]

**Review by jmcadam — COMMENTED** [review]
Ignore ES comment

File: src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/elasticsearch.scala (hunk 2 above)
Comment (jonsoloway-hmcts) [reply src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/api/elasticsearch.scala:1]: It's still there, just been edited.

**Pull request merged**